app/views/reviews/edit.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= label_tag :title %>
     <%= text_field_tag :title, @review.title%><br>
     <%= label_tag :rating %>
-    <%= text_field_tag :rating, @review.rating %><br>
+    <%= select_tag :rating, options_for_select([1, 2, 3, 4, 5], @review.rating) %><br>
```

Comment:
Changing so the user can only select a 1 - 5 integer. The second argument of @review.rating pre-populates with the original rating.